tests/test_dryrun.py

```diff
@@ -41,16 +60,96 @@ def test_get_referenced_tables_empty(self, tmp_path):
         dryrun = DryRun(str(query_file))
         assert dryrun.get_referenced_tables() == []
 
+    def test_get_sql(self, tmp_path):
+        os.makedirs(tmp_path / "telmetry_derived")
+        query_file = tmp_path / "telmetry_derived" / "query.sql"
+
+        sql_content = "SELECT 123 "
+        query_file.write_text(sql_content)
+
+        assert DryRun(sqlfile=str(query_file)).get_sql() == sql_content
+        with pytest.raises(ValueError):
+            DryRun(sqlfile="invalid path").get_sql()
+
     def test_get_referenced_tables(self, tmp_path):
         os.makedirs(tmp_path / "telmetry_derived")
         query_file = tmp_path / "telmetry_derived" / "query.sql"
         query_file.write_text(
             "SELECT * FROM telemetry_derived.clients_daily_v6 "
             "WHERE submission_date = '2020-01-01'"
         )
-        dryrun = DryRun(str(query_file))
-        response = dryrun.get_referenced_tables()
+        view_file = tmp_path / "telmetry_derived" / "view.sql"
+        view_file.write_text(
+            """
+            CREATE OR REPLACE VIEW
+            `moz-fx-data-shared-prod.telemetry.clients_daily`
+            AS
+            SELECT
+            *
+            FROM
+            `moz-fx-data-shared-prod.telemetry_derived.clients_daily_v6`
+        """
+        )
+        query_dryrun = DryRun(str(query_file)).get_referenced_tables()
+        view_dryrun = DryRun(str(view_file), strip_dml=True).get_referenced_tables()
 
-        assert len(response) == 1
-        assert response[0]["datasetId"] == "telemetry_derived"
-        assert response[0]["tableId"] == "clients_daily_v6"
+        assert len(query_dryrun) == 1
```

Comment:
Can you move the test setup directly before the asserts? This is mostly a matter of style, but I think it's useful to group tests as if you were to put each case into a function.

Reply:
Thank you! This is now fixed.